plugins/modules/azure_rm_storageaccount.py

```diff
@@ -94,6 +94,50 @@
             -  Allows https traffic only to storage service when set to C(true).
         type: bool
         version_added: "2.8"
+    network_acls:
+        description:
+            - Manages the Firewall and virtual networks settings of the storage account.
+        type: dict
+        version_added: "2.10"
+        suboptions:
+            default_action:
+                description:
+                    - Default firewall traffic rule.
+                    - If set to 'Allow' no other settings have effect.
+                choices:
+                    - Allow
+                    - Deny
+                default: Allow
+            bypass:
+                description:
+                    - When C(default_action) is set to 'Deny' this controls which Azure components can still reach the Storage Account.
+                    - The list is comma separated.
+                    - It can be any combination of the following: AzureServices, Logging, Metrics.
```

Comment:
```suggestion
                    - It can be any combination of the following: C(AzureServices), C(Logging), C(Metrics).
```

Reply:
@Fred-sun your comment is valid but I'm having second thoughts on keeping this on the Ansible side a string. Would it be more Ansible like to make this a list and convert it to comma separated string for the Azure api?